content/3_Silver/DFS.mdx

```diff
@@ -585,3 +585,40 @@ public class BuildingTeams
 ### Problems
 
 <Problems problems="bip" hideSuggestProblemButton />
+
+## Quiz
+Other questions concerning DFS and trees can be found in the Introduction to Tree Algorithms section.
+
+<Quiz>
+	<Quiz.Question>
+		Can a DFS with adjacency list be used to find shortest paths in linear time?
+		<Quiz.Answer>
+			Yes
+			<Quiz.Explanation>
+				Incorrect.
```

Comment:
I feel like the correct explanation could be pasted here.

Reply:
(from the correct answer)

Reply:
benq told me not to